examples/mobile/UIComponents/components/controllers/radio.html

```diff
@@ -17,46 +17,38 @@ <h1>
 			</h1>
 		</div>
 		<div class="ui-content">
-			<ul class="ui-listview">
-				<li class="ui-li-flex">
-					<span class="ui-li-area ui-li-area-a">
-						<span class="ui-li-text">
-							<span id="radio-text-1">
-								Radio on
-							</span>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider ui-li-has-radio">
+					<input checked="checked" id="radio-1" name="radio-group" type="radio" />
```

Comment:
The checkbox uses the "checkboxes" class.
Doesn't radio use class?

Reply:
It is used in the checkbox.js